index.html

```diff
@@ -10,6 +10,12 @@
     out in the same tree and use relative links so that they'll work offline.
    -->
 
+  <script src="https://www.w3.org/scripts/jquery/2.2/jquery.min.js"  crossorigin="anonymous"></script>
+
+  <link rel="stylesheet" type="text/css" href="https://www.w3.org/scripts/jquery-datatables/1.10/css/jquery.dataTables.css">
+  
+  <script type="text/javascript" charset="utf8" src="https://www.w3.org/scripts/jquery-datatables/1.10/js/jquery.dataTables.js"></script>
```

Comment:
@deniak - Apparently the 1.10 and 2.2 directories don't symlink to the latest existing sub-version-directory (try browsing through all the "latest patch" links from https://www.w3.org/scripts/ yourself), so neither jquery nor datatables work yet... so @jricher -- I'm suggesting a test with the more-specific links below, which we can change back to *these* links when SysTeam gets back to us.

```suggestion
  <script src="https://www.w3.org/scripts/jquery/2.2.4/jquery.min.js"  crossorigin="anonymous"></script>

  <link rel="stylesheet" type="text/css" href="https://www.w3.org/scripts/jquery-datatables/1.10.24/css/jquery.dataTables.css">
  
  <script type="text/javascript" charset="utf8" src="https://www.w3.org/scripts/jquery-datatables/1.10.24/js/jquery.dataTables.js"></script>
```

Reply:
@deniak -- Ugh. Please note the different behavior of the "latest patch" links at https://www.w3.org/scripts -- just start at the top...

Reply:
@deniak -- Major problem with the current behavior is that there's no way to discover where to link, neither by looking at, nor by drilling down through, that `scripts` index page.

Reply:
@deniak -- Also please note the CORS issue, flagged above by @jricher.

Reply:
I believe I understand the issue now. There's probably a confusion between the column 'Available versions' and 'Browse files'. You should actually use the links in the last column.
https://www.w3.org/scripts/jquery-datatables/1.10/ will redirect to https://www.w3.org/scripts/jquery-datatables/1.10.24/
and
https://www.w3.org/scripts/jquery/2.2/ to https://www.w3.org/scripts/jquery/2.2.4/

I reckon the UI should probably be reworked.

Reply:
> @deniak -- Also please note the CORS issue, flagged above by @jricher.

Yes, these scripts are mostly for w3.org resources but I'll discuss with the other systeamers to see if we can relax that policy a little bit.

Reply:
@deniak - "You should actually use the links in the last column" -- I had tried that, and got 404s, hence my earlier comments about the lack of symlinks.  Perhaps I was over-eager and clicked before your/SysTeam's process had completed.

Reply:
Sorry it took so long but finally discussed that issue with systeam and we ended up opening CORS for a few scripts under /scripts, mainly, jQuery, jQuery UI, jQuery datatables and MathJax.
You should now be able to use these scripts from github.io.

Reply:
@jricher -- My latest suggestions need to be applied to your PR.  Once they are, the preview should work, with sorts and table size and all.

@OR13 -- Next review should happen once @jricher applies the changes above.

Reply:
@TallTed thanks for the heads up :)

Reply:
I just folded in the changes, now we're not getting CORS errors on the datatables but we're still getting CORS errors on jquery.